### ISSUE TITLE
Update TransformOptions to allow enableImplicitConversion setting

### ISSUE THREAD

**Issue by FSM1** [nestjs/nest#2253]
## I'm submitting a...

<pre><code>
[ ] Regression 
[ ] Bug report
[x] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

Setting the newly added `enableImplicitConversion` flag causes a TS error.

```
  app.useGlobalPipes(new ValidationPipe({
    whitelist: true,
    transform: true,
    transformOptions: {
       enableImplicitConversion: true,
    }
  }));
```
## Expected behavior

The setting should be allowed.

## Environment

<pre><code>
Nest version: 6.0.4

 
For Tooling issues:
- Node version: 10.13.0  
- Platform: Windows 

Others:

</code></pre>

**Comment by kamilmysliwiec** [comment]
This option has been introduced in the latest version of `class-transformer` package. Do you want to create a PR for this enhancement?

**Comment by FSM1** [comment]
thats is exactly the plan @kamilmysliwiec 

I will submit the PR for this  shortly.

**Comment by kamilmysliwiec** [comment]
Published as 6.3.0
